lerna.json

```diff
@@ -3,9 +3,19 @@
 	"version": "independent",
 	"command": {
 		"publish": {
-			"ignoreChanges": ["*.md"],
-			"message": "chore(release): publish"
+			"ignoreChanges": ["**/*.md", "**/__tests__/**"],
+			"message": "chore(release): publish",
+			"conventionalCommits": true,
+			"push": true
+		},
+		"version": {
+			"allowBranch": "main",
+			"conventionalCommits": true,
+			"push": true,
+			"createRelease": "github"
```

Comment:
Also not a review comment but be wary that for this to work you need to set a `GH_TOKEN` env var in your local system which has to be a GH PAT. Just something to share with anyone who will be doing releases.